### PR TITLE
Prepare for 2.0

### PR DESCRIPTION
**Pull request by OsirisTerje**
I think all is done for merging this in now.  
The only things remaining is to switch the object model back to 2012, but that should be the last step before release. 

Need to run it through some more extensive testing too.

### CODE REVIEW

**Comment by CharliePoole** [comment]
This looks good for merging to me.

**Comment by CharliePoole** [comment]
Please merge and release when ready.

File: src/NUnitTestAdapter/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -21,5 +21,5 @@
 [assembly: ComVisible(false)]
 
 [assembly: Guid("c0aad5e4-b486-49bc-b3e8-31e01be6fefe")]
-[assembly: AssemblyVersion("1.2.0.1")]
-[assembly: AssemblyFileVersion("1.2.0.1")]
+[assembly: AssemblyVersion("1.9.9.0")]
```
Comment (rprouse) [inline src/NUnitTestAdapter/Properties/AssemblyInfo.cs]: Should this be 2.0.0.0 based on the package name?

File: src/NUnitTestAdapterInstall/source.extension.vsixmanifest [hunk 2]
```diff
@@ -3,10 +3,10 @@
   <Identifier Id="NUnitTestAdapterInstall..7c53286e-ac4f-485f-915c-5ec5a4c47b0c">
     <Name>NUnit Test Adapter</Name>
     <Author>Charlie Poole</Author>
-    <Version>1.2.0.1</Version>
-    <Description xml:space="preserve">NUnit adapter for integrated test execution under Visual Studio 2012 (all updates), Visual Studio 2013 (all updates), and the Visual Studio 2015 Preview and CTPs. Compatible with NUnit 2.0 through 2.6.3.</Description>
+    <Version>1.9.9.2</Version>
```
Comment (rprouse) [inline src/NUnitTestAdapterInstall/source.extension.vsixmanifest]: Should this also be 2.0.0.0?

**Comment by rprouse** [comment]
Other than the versions, I have tested and everything seems good to me.

File: src/NUnitTestAdapter/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (OsirisTerje) [inline src/NUnitTestAdapter/Properties/AssemblyInfo.cs]: Yes it should. 
I just kept it at 1.9.9 while beta'ing it, as vsix'es dont have any beta concept like nuget packages has.

File: src/NUnitTestAdapterInstall/source.extension.vsixmanifest (hunk 2 above)
Comment (OsirisTerje) [inline src/NUnitTestAdapterInstall/source.extension.vsixmanifest]: Yes.  This manifest file looks much older, the Description seems outdated.  Sure this is a recent one ?

File: src/NUnitTestAdapter/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (CharliePoole) [inline src/NUnitTestAdapter/Properties/AssemblyInfo.cs]: A possible idea for the future would be to set up an entirely separate
package for development versions.

**Pull request merged**